app/helpers/application_helper.rb

```diff
@@ -28,4 +28,18 @@ def md(markdown_in)
     markdown = Redcarpet::Markdown.new(Redcarpet::Render::HTML, extensions = {:fenced_code_blocks => true})
     markdown.render(markdown_in).html_safe
   end
+
+  def bootstrap_class_for(flash_type)
+    bootstrap_classes.fetch(flash_type.to_sym, custom_flash(flash_type))
```

Comment:
Here's why the reason for the flashes being passed in as strings 
http://blog.bigbinary.com/2014/07/24/flash-access-changes-rails-4-1.html